src/components/Dashboard/SortList/SortList.scss

```diff
@@ -0,0 +1,43 @@
+.btn{
```

Comment:
I think it was made clear several times, we use ident with 2 spaces, not four.

Reply:
Sorry i didn't notice.
Usually, linter highlights problems with ident, so I did not pay attention.